### PR TITLE
2119 new menu item gdt

### LINKED ISSUE #2119: 🚀 [Feature] New menu item GDT
## 🚀 Feature
Can we show another item "GDT" below the menu item "Transactions" (if GDT exists), which leads directly to the display of the GDT?

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
there is now a separate router entry for transaction and gradido transform. 

### Issues
- fixes #2119 

 

https://user-images.githubusercontent.com/1324583/183288276-f7a96dac-b2fd-4c7d-8db8-d2eedfdf9d8f.mp4

### CODE REVIEW

File: frontend/src/pages/Transactions.spec.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import { mount } from '@vue/test-utils'
 import Transactions from './Transactions'
 import { GdtEntryType } from '@/graphql/enums'
+import flushPromises from 'flush-promises'
```
Comment (Mogge) [inline frontend/src/pages/Transactions.spec.js:4]: Why?

**Review by Mogge — CHANGES_REQUESTED** [review]
Wow. What a hack.

File: frontend/src/pages/Transactions.spec.js (hunk 1 above)
Comment (ogerly) [reply frontend/src/pages/Transactions.spec.js:4]: flushPromises deleted

File: frontend/src/pages/Transactions.spec.js [hunk 2]
```diff
@@ -158,6 +163,7 @@ describe('Transactions', () => {
 
           describe('click on GDD tab', () => {
             beforeEach(() => {
+              flushPromises()
```
[the remark this answers is not in the document]
Comment (ogerly) [reply frontend/src/pages/Transactions.spec.js:166]: flushPromises deleted

**Comment by ogerly** [comment]
> 

it would be conceivable to remove the tab system and create separate pages for each ...

File: frontend/src/components/GddSend/TransactionConfirmationLink.spec.js [hunk 3]
```diff
@@ -46,5 +46,17 @@ describe('GddSend confirm', () => {
...
     })
+
+    describe('has totalBalance under 0', () => {
+      beforeEach(async () => {
+        await wrapper.setProps({
+          balance: 0,
+        })
+      })
+
+      it('has button disable', () => {
```
Comment (Mogge) [inline frontend/src/components/GddSend/TransactionConfirmationLink.spec.js:57]:
```suggestion
      it('has send button disabled', () => {
```

File: frontend/src/components/GddSend/TransactionConfirmationLink.spec.js [hunk 4]
```diff
@@ -46,5 +46,17 @@ describe('GddSend confirm', () => {
         expect(wrapper.findAll('div.confirm-box-link').at(0).exists()).toBeTruthy()
       })
     })
+
+    describe('has totalBalance under 0', () => {
```
Comment (Mogge) [inline frontend/src/components/GddSend/TransactionConfirmationLink.spec.js:50]:
```suggestion
    describe('has total balance equal 0', () => {
```

**Review by Mogge — CHANGES_REQUESTED** [review]

**Review by Tirokk — APPROVED** [review]
Hey @ogerly 
works fine!

Great 🚀💫
I approve!

**Review by Tirokk — APPROVED** [review]
Now I approve again.

**Pull request merged**